src/components/organisms/AssetActions/Compute/index.tsx

```diff
@@ -491,7 +491,9 @@ export default function Compute({
             action={<SuccessAction />}
           />
         )}
-        <Web3Feedback isBalanceSufficient={isBalanceSufficient} />
+        {ddo.service[1].attributes.main.privacy.allowNetworkAccess && (
```

Comment:
not sure this is the right check. Technically an algorithm is private if it has a `compute` service instead of an `access` service

Reply:
since we are in `compute` component I think just checking if it is an algorithm is enough

Reply:
I see, it's clear now, thanks! Now the checks should be correct.

Reply:
> since we are in `compute` component I think just checking if it is an algorithm is enough

Think you're right, I removed the redundant check.